features/admin.extensions.v1/configs/server-configuration.tsx

```diff
@@ -47,6 +49,257 @@ import {
 import { generatePasswordHistoryCount } from "../components/password-history-count/components";
 import { updatePasswordPolicyProperties } from "../components/password-policies/api/password-policies";
 
+const ServerConfigurationsConstants: any = {
```

Comment:
Is this expected?

Reply:
Fixed with [fdb9482](https://github.com/wso2/identity-apps/pull/6225/commits/fdb9482d54afa818112f2f9770bd342c86867a7b)